src/Jackett/Content/index.html

```diff
@@ -1,8 +1,23 @@
-﻿<!DOCTYPE html>
+<!DOCTYPE html>
 
 <html lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta name="mobile-web-app-capable" content="yes"/>
+    <meta name="apple-mobile-web-app-capable" content="yes"/>
     <meta charset="utf-8" />
+	<link rel="apple-touch-icon" href="../apple-touch-icon.png"/>

```

Comment:
seems like tabs and spaces have been merged, I suggest that you use 4 spaces instead of a tab